### PR TITLE
Scroll highlighted item into view

### PR DESCRIPTION
**Pull request by josephschmitt**
If the element being selected is off screen (or cut off by overflow hidden), this will scroll it so it's visible.

### CODE REVIEW

File: src/angularComponent/ngcOmniboxController.js [hunk 1]
```diff
@@ -137,6 +137,21 @@ export default class NgcOmniboxController {
     } else {
       this._highlightedItem = null;
     }
+
+    // Wait until next render cycle
+    setTimeout(() => {
+      const selectedEl = this.element.querySelector('[aria-selected]');
+
+      if (selectedEl) {
+        if (selectedEl.scrollIntoView) {
```
Comment (drawyan) [inline src/angularComponent/ngcOmniboxController.js]: typeof selectedEl.scrollIntoView === 'function' ?

File: src/angularComponent/ngcOmniboxController.js [hunk 2]
```diff
@@ -137,6 +137,21 @@ export default class NgcOmniboxController {
...
+          // Standard way
+          selectedEl.scrollIntoView();
+        } else if (selectedEl.scrollIntoViewIfNeeded) {
```
Comment (drawyan) [inline src/angularComponent/ngcOmniboxController.js]: Same as above

File: src/angularComponent/ngcOmniboxController.js (hunk 1 above)
Comment (josephschmitt) [inline src/angularComponent/ngcOmniboxController.js]: Done

File: src/angularComponent/ngcOmniboxController.js (hunk 2 above)
Comment (josephschmitt) [inline src/angularComponent/ngcOmniboxController.js]: Done

**Pull request merged**